docs/src/main/sphinx/client/cli.rst

```diff
@@ -232,14 +232,16 @@ certificate usage:
     - The password for the keystore. This must match the password you specified
       when creating the keystore.
   * - ``--keystore-type``
-    - Keystore type.
+    - Determined by keystore file formant. Common keystore types are JKS,
+      JCEKS, and PKCS#12.
```

Comment:
Valid types are documented here: https://docs.oracle.com/en/java/javase/17/docs/specs/security/standard-names.html#keystore-types

Note that you can't use a hash in `PKCS#12`, in a terminal this would at least require quoting since it's a special character in bash.

Reply:
yeah .. I asked @Jessie212 to add the link already

Reply:
@mosabua I was just going to tell Jas that I'd forgotten to include that very link. 

Also, @nineinchnick, what about the truststore types? I didn't come across much on truststores during my research. Are there as many types as with keystore types? And am I wrong about PEM as a type? Should it be JSSE instead?

Reply:
trust store are the same from all I know

Reply:
Yes, the trust store is a kind of key store. The difference is that the trust store is used to read (CA) certificates used to verify (_trust_) the remote server's certificate and the key store is used to read the client certificate used to authenticate with the remote server (sometimes called mutual TLS).

Ref: https://github.com/trinodb/trino/blob/52e4e3eaf267dead0af26757f91790768ba3ef68/client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java#L268

Reply:
I also wonder if it would make sense to mention what is the default value here, because these parameters are optional. Usually, it is `jks`, but can be set using a security property in the Java runtime.

Ref: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/security/KeyStore.html#getDefaultType()